src/mapping/config/MappingConfiguration.cpp

```diff
@@ -228,12 +233,20 @@ MappingConfiguration::MappingConfiguration(
   auto projectToInput = XMLAttribute<bool>(ATTR_PROJECT_TO_INPUT, true)
                             .setDocumentation("If enabled, places the cluster centers at the closest vertex of the input mesh. Should be enabled in case of non-uniform point distributions such as for shell structures.");
 
+  auto attrGeoMultiscaleType = XMLAttribute<std::string>(ATTR_GEOMETRIC_MULTISCALE_TYPE)
+                                   .setDocumentation("Type of geometric multiscale mapping. Either 'spread' or 'collect'.");
+  auto attrGeoMultiscaleAxis = XMLAttribute<std::string>(ATTR_GEOMETRIC_MULTISCALE_AXIS)
```

Comment:
Indeed, that's already in the wish-list: https://github.com/precice/precice/issues/1726

But since this feature is already so niche, we can go a long way just with the x/y/z assumption.